vector/src/main/res/drawable/ic_pro_layout.xml

```diff
@@ -0,0 +1,84 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
rename the two drawable vectors to ensure they have a common prefix: `ic_layout_pro` and `ic_layout_simple`. Also those are not really icon, so maybe `setting_layout_pro` and `setting_layout_simple` would be better names